solutions/fscloud/main.tf

```diff
@@ -0,0 +1,22 @@
+module "resource_group" {
+  count                        = (var.resource_group_id == null) ? 1 : 0
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.0"
+  resource_group_name          = var.existing_resource_group == false ? var.resource_group_name : null
+  existing_resource_group_name = var.existing_resource_group == true ? var.resource_group_name : null
+}
+
+module "cloudant" {
+  source              = "../../modules/fscloud"
+  resource_group_id   = var.resource_group_id == null ? module.resource_group[0].resource_group_id : var.resource_group_id
```

Comment:
Remove the count ad per https://github.com/terraform-ibm-modules/terraform-ibm-cloudant/pull/32/files#r1392398763 and always grab this value from the module

Reply:
If I have to remove `count` then the `resource_group_id` variable will be removed too. That means the User has only 2 options - either to pass the existing resource group name or not pass anything (in that case, it will create a new resource group). There will not be any provision to pass "resource group ID" anymore. If we are okay with this design I will implement the changes.

Reply:
yep - thats what we want